packages/firebase_ml_vision/example/pubspec.yaml

```diff
@@ -26,7 +26,6 @@ dependency_overrides:
     path: ../../firebase_core/firebase_core_web
 
 dev_dependencies:
-  e2e: ^0.6.1
```

Comment:
This change isn't a complete removal of e2e, there's still usages here:  https://github.com/FirebaseExtended/flutterfire/blob/master/packages/firebase_ml_vision/example/android/app/src/androidTest/java/io/flutter/plugins/firebasemlvision/EmbeddingV1ActivityTest.java

Reply:
Thanks for pointing me there! Reverted the change.

I naively ran 
```
flutter drive test_driver/firebase_ml_vision.dart
```
and it worked without e2e, so I assumed it's not necessary. But that's better left to another PR anyway....